src/dom.js

```diff
@@ -253,6 +289,44 @@ export function childElementByAttr(parent, attr) {
 
 
 /**
+ * Finds the last child element that has the specified attribute.
+ * @param {!Element} parent
+ * @param {string} attr
+ * @return {?Element}
+ */
+export function lastChildElementByAttr(parent, attr) {
+  return lastChildElement(parent, el => {
+    if (!el.hasAttribute(attr)) {
+      return false;
+    }
+    return true;
+  });
+}
+
+
+/**
+ * Finds all child elements that has the specified attribute.
+ * @param {!Element} parent
+ * @param {string} attr
+ * @return {Array.<Element>|NodeList}
```

Comment:
I can do that. I asked about this during implementation and the answer I got that it's ok, but I'll copy the NodeList into an Array.